### ISSUE TITLE
app.send preventdefault by default

### ISSUE THREAD

**Issue by yoshuawuyts** [sethvincent/virtual-app#8]
This will trigger the default action of reloading the page when submit is triggered:

``` js
hx`
  <form onsubmit=app.send({foo: 'bar'})>
    <input id="new-todo" placeholder=${placeholder}>
    <input type="submit" style=${{ display: 'none' }}>
  </form>
`
```

It'd be nice to call `e.preventDefault()` always, and add a switch to enable it if needed. We're already handling it so there generally shouldn't be a reason to call it.

``` js
// preventDefault is called
hx`
  <form onsubmit=app.send({foo: 'bar'})>
    <input id="new-todo" placeholder=${placeholder}>
    <input type="submit" style=${{ display: 'none' }}>
  </form>
`
```

``` js
// preventDefault is not called
hx`
  <form onsubmit=app.send({foo: 'bar'}, false)>
    <input id="new-todo" placeholder=${placeholder}>
    <input type="submit" style=${{ display: 'none' }}>
  </form>
`
```

What do you reckon?

**Comment by sethvincent** [comment]
This looks really useful for simplifying event handlers with hyperx. :+1:

**Comment by sethvincent** [comment]
closed by #10
